### PR TITLE
Add Cols Class

### PR DESCRIPTION
**Pull request by Arunsp2000**
To test this class for the time being, please uncomment lines 54 to 77 and run:
**_python -m src.cols_**

### CODE REVIEW

File: src/cols.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+        '''
+        First, storing all the Num and Sym Objects into the "all" attribute
+        Second, performing regex to separate the dependent from independent variables into the attributes "y" and "x"
+        respectively
+        For the atributes "x" and "y" we skip all columns which end with ":".
+        '''
+        for i in range(len(names)):
+            if(re.search("\*[A-Z]", names[i])):
+                self.all.append(Num(i, names[i]))
```
Comment (nitesh31mishra) [inline src/cols.py]: Can you check in the code file of Lua, over there they are using the push function which they have written manually. So are we supposed to use the append function? Or do we have to implement a push function?

Comment (Arunsp2000) [reply src/cols.py]: Im guessing we can use this?
In LUA we have only one kind of list(dictionary basically). The push function is just using array indexing so i think it should be fine.

Comment (nitesh31mishra) [reply src/cols.py]: Okay!!

File: src/cols.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        self.y = []
+
+        '''
+        First, storing all the Num and Sym Objects into the "all" attribute
+        Second, performing regex to separate the dependent from independent variables into the attributes "y" and "x"
+        respectively
+        For the atributes "x" and "y" we skip all columns which end with ":".
+        '''
+        for i in range(len(names)):
+            if(re.search("\*[A-Z]", names[i])):
```
Comment (agupta15k) [inline src/cols.py]: This doesn't appear to be matching what lua code is doing. Here, the code is matching column names which have a wildcard followed by an uppercase alphabet, while the lua code is matching the column names which are starting with an upper case alphabet. Correct me if I misunderstood this.

File: src/cols.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+#     print("-"*100)
+#     for i in cols.y:
+#         print(i.name)
+#     print("-"*100)
+#     print(cols.names)
+#     print(cols.klass)
+
+            
+
+# showStats()
```
Comment (agupta15k) [inline src/cols.py]: No new line at the end of file

File: src/cols.py (hunk 2 above)
Comment (Arunsp2000) [reply src/cols.py]: On it. It is a mistake. My bad!

**Review by agupta15k — APPROVED** [review]
Approved

**Pull request merged**